vars/tmsExport.groovy

```diff
@@ -0,0 +1,55 @@
+import groovy.transform.Field
+import com.sap.piper.JenkinsUtils
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/tmsExport.yaml'
+@Field Set GENERAL_CONFIG_KEYS = [
```

Comment:
Is it really necessary to define all these config keys (params) here in the wrapper groovy file?

Reply:
Groovy / unit and Documentation / Build tests are failing if parameters are not present:
https://github.com/SAP/jenkins-library/actions/runs/3930914140/jobs/6721498428
https://github.com/SAP/jenkins-library/actions/runs/3930914161/jobs/6721498153
Failed tests: 
CommonStepsTest.generalConfigKeysSetPresentTest(CommonStepsTest)
  Run 1: CommonStepsTest.generalConfigKeysSetPresentTest:239 Steps without GENERAL_CONFIG_KEYS field (or that field is not a Set): [tmsExport]
CommonStepsTest.parametersKeysSetPresentTest(CommonStepsTest)
  Run 1: CommonStepsTest.parametersKeysSetPresentTest:259 Steps without PARAMETER_KEYS field (or that field is not a Set): [tmsExport]
CommonStepsTest.stepConfigKeysSetPresentTest(CommonStepsTest)
  Run 1: CommonStepsTest.stepConfigKeysSetPresentTest:249 Steps without STEP_CONFIG_KEYS field (or that field is not a Set): [tmsExport]

Reply:
These parameters are not necessary as this is a pure go step. Check [any other](https://github.com/SAP/jenkins-library/blob/master/vars/abapEnvironmentPullGitRepo.groovy) such step...

Reply:
2 parameters are defined in your example. How can you comment failing unit tests?

Reply:
Have just removed config specific fields and Groovy / unit tests failing again, see https://github.com/SAP/jenkins-library/actions/runs/4415998403/jobs/7739701947